client/gutenberg/extensions/simple-payments/product-placeholder.jsx

```diff
@@ -9,53 +9,48 @@ import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
  * Internal dependencies
  */
 import './product-placeholder.scss';
+import paypalImage from './paypal-button.png';
+import paypalImage2x from './paypal-button@2x.png';
```

Comment:
😍

Reply:
Could this be something that is handled via the build script? 
Creating 2x images.

Reply:
Good idea — definitely material for the backlog.

Reply:
We don't want to be creating 2x images from 1x 😉 . Anything automated would preferably scale _down_.